### DISCUSSION TITLE
Question: Any way possible to access Gmail labels

### DISCUSSION THREAD

**Discussion by Vaibhav2608** [InboxSDK/InboxSDK#922] [General]
**Question(s)**
I want to give a button to add labels to the current thread or conversation with a button click. I tried to find it in the doc but I need help finding it. Can anyone help

**Comment by 1TheMuffinMan** [comment]
You can add display labels like this https://inboxsdk.github.io/inboxsdk-docs/lists/#addlabellabeldescriptor but they won't persist. If you truly want to configure a new label, add a filter etc you'd have to request that scope and do it with an API call using the Gmail API.
